apps/cnspec/cmd/serve.go

```diff
@@ -83,6 +83,10 @@ var serveCmd = &cobra.Command{
 			hc := backgroundjob.NewHealthPinger(ctx, client.HttpClient, client.ApiEndpoint, 5*time.Minute)
 			hc.Start()
 			defer hc.Stop()
+
+			checkin := backgroundjob.NewCheckinPinger(ctx, client.HttpClient, client.ApiEndpoint, conf.runtime.UpstreamConfig, 2*time.Hour)
```

Comment:
We do we need health and checkin?

Reply:
The health check only checks the availability of the upstream API. No data is updated upstream, but it is also used for clock skew detection.

The check-in reports the current version and other data and this is updated upstream.

We could combine this into one API call. I separated it because of the clock skew detection and this way we can use different intervals.

Reply:
I think we should keep the checkin call and not do the health check. I like the separation but I do not see any benefit of doing the health check in addition to the checkin.

Reply:
:+1: Changed it.